src/sass/components/_org-cards.scss

```diff
@@ -10,23 +10,51 @@
 
 .nested-card {
   .slds-card__header {
+    --card-header-toggle-space: 0;
+    --dropdown-right: 0;
     padding-left: 0;
-    padding-right: 0;
-  }
 
-  .slds-media {
-    margin-bottom: $spacing-medium;
+    .has-org & {
+      margin-right: $spacing-medium;
+    }
   }
 }
 
 // override min-width and force header to wrap on nested card headers
 .wrap-inner-truncate {
   .slds-card__header {
+    align-items: baseline;
     flex-wrap: wrap;
+    // hold space for absolutely positioned dropdown toggle
+    padding-right: var(--card-header-toggle-space, #{$spacing-x-large});
+
+    // Specifically target OrgActions in card header
+    .slds-no-flex {
+      align-items: baseline;
+      display: flex;
```

Comment:
Adding `display: flex` to `.slds-no-flex` feels pretty weird to me. Is there any way we can add a new class for this override - or remove that one?

Reply:
@mirisuzanne It feels gross to me too, and I know we don't like to change the meaning of one of their classes, but we are at least nesting it in our own `.wrap-inner-truncate` class if that helps a tiny bit. @jgerigmeyer correct me if I am wrong on this, but I don't think we have access to the element with the `slds-no-flex` class.

Reply:
> @jgerigmeyer correct me if I am wrong on this, but I don't think we have access to the element with the `slds-no-flex` class.

That's correct.

Reply:
ok, yeah, probably nothing better do do about it then…